scripts/fuzz_opt.py

```diff
@@ -1402,6 +1388,8 @@ def handle(self, wasm):
 lit_tests = shared.get_tests(shared.get_test_dir('lit'), test_suffixes, recursive=True)
 all_tests = core_tests + passes_tests + spec_tests + wasm2js_tests + lld_tests + unit_tests + lit_tests
 
+fuzz_cases = shared.get_tests(shared.get_test_dir('fuzz'), test_suffixes, recursive=True)
```

Comment:
This won't break if the `fuzz` directory doesn't exist, will it?

Reply:
Good point, I'll add a readme.txt in that directory to force it to exist in git.